### PR TITLE
feat(website): rework the website hero + animated taiko image

### PR DESCRIPTION
**Pull request by wolfderechter**
This is my take on issue #399. I also added an animation on the TaikoGeom image upon scroll, would love to hear thoughts on this but can always be removed. See the picture for how it looks on an Iphone 11 Pro.
Iphone11Pro

### CODE REVIEW

File: packages/website/src/components/Hero/index.tsx [hunk 1]
```diff
@@ -1,32 +1,67 @@
...
+    } else {
+      taikoGeomParent.style.width = `${
+        window.innerWidth.valueOf() / 2 + elementHeight * 9
+      }px`;
+
+      // Lower the TaikoGeom opacity on scroll
+      taikoGeom.style.opacity = `${(1 - elementHeight / 250) * 0.8}`;
+    }
+  };
+  window.addEventListener("scroll", changeTaikoGeom);
```
Comment (WormydogDOTeth) [inline packages/website/src/components/Hero/index.tsx:32]: Does this need to ever be removed at any point?

Comment (wolfderechter) [reply packages/website/src/components/Hero/index.tsx:32]: This is the animation feature on the taiko geometry image that I played around with, I like the outcome but if others don't like it that section could be removed

Comment (WormydogDOTeth) [reply packages/website/src/components/Hero/index.tsx:32]: Actually to me it looks good but was just wondering about the listener, usually memory leaks occur if it isn't removed but not sure about that with svelte.

**Comment by d1onys1us** [comment]
i am seeing a vercel error @wolfderechter , do you know why? this looks great at least from the screenshot, thank you for your contribution! ❤️‍🔥 

`ReferenceError: window is not defined`

**Comment by wolfderechter** [comment]
There was a problem with the window usage, resolved now

**Comment by d1onys1us** [comment]
> There was a problem with the window usage, resolved now

this looks awesome, thank you again @wolfderechter ! two small comments:
1: is it possible to brighten the hue / saturation in the default view (scrolled all the way to the top)
brighter/more saturated before:
image
less saturated after:
image

2. can we put this behind the text? it looks like on mobile view that it goes "in front". also is it possible to make it any smaller on the mobile view / out of the way? 
image

File: packages/website/src/components/Hero/index.tsx (hunk 1 above)
Comment (d1onys1us) [reply packages/website/src/components/Hero/index.tsx]: this is not using svelte fyi, it is using react. (we use svelte on our Bridge UI, this is docusaurus + React). @WormydogDOTeth

**Comment by WormydogDOTeth** [comment]
Noted! Digesting it all, thanks for that!

**Review by dionysuzx — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Comment by d1onys1us** [comment]
hi @wolfderechter I noticed there is a gap / overflow on the right side, i'm only noticing this on mobile view. do you know what the fix for it is?

## screenshot
image

**Comment by wolfderechter** [comment]
> hi @wolfderechter I noticed there is a gap / overflow on the right side, i'm only noticing this on mobile view. do you know what the fix for it is?
> ## screenshot
> 
> image

Oh snap, you're right. Just fixed it on #443, there was still an overflow to the right that I overlooked

**Comment by d1onys1us** [comment]
> > hi @wolfderechter I noticed there is a gap / overflow on the right side, i'm only noticing this on mobile view. do you know what the fix for it is?
> > ## screenshot
> > image
> 
> Oh snap, you're right. Just fixed it on #443, there was still an overflow to the right that I overlooked

looks like there is a merge conflict @wolfderechter

**Pull request merged**